.eslintrc

```diff
@@ -4,5 +4,8 @@
   ],
   "plugins": [
     "prettier"
-  ]
+  ],
+  "rules": {
+    "prettier/prettier": "error"
```

Comment:
@jquense explicitly took this out earlier, which is why only ci-lint errors on this. do we want it back?